server/runtime/src/main/java/org/infinispan/server/logging/annotation/Logged.java

```diff
@@ -0,0 +1,24 @@
+package org.infinispan.server.logging.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation should be used on methods that need to be notified when information is logged by the
+ * {@link org.infinispan.util.logging.events.EventLogger}. There is no distinction between the log level or category.
```

Comment:
I think `EventLoggerNotifier`, and `Logged` should be moved to `core`. The `BasicEventLogger` and other `EventLogger` impls can then just call `notifyLog` as required as there's no cache to listen to.